src/sentry/tagstore/base.py

```diff
@@ -314,6 +316,6 @@ def get_group_tag_keys_and_top_values(
         return tag_keys
 
     def get_group_seen_values_for_environments(
```

Comment:
Same comment as for `get_groups_user_counts`

Reply:
added this too